pytest_mypy_plugins/item.py

```diff
@@ -224,8 +225,11 @@ def typecheck_in_same_process(
             # add current directory to path
             sys.path.insert(0, str(execution_path))
 
-            with capture_std_streams() as (stdout, stderr):
-                return_code = run_mypy_typechecking(mypy_cmd_options)
+            stdout = io.StringIO()
+            stderr = io.StringIO()
+
+            with closing(stdout), closing(stderr):
```

Comment:
```suggestion
            with stdout, stderr:
```

Reply:
🤦 somehow missed that closing is no longer needed in Py3.

I just noticed that this code is still wrong, though not covered by the tests because you cannot call .getvalue() on a closed buffer.